litmus-portal/frontend/src/views/Home/AgentConfiguredHome/RecentWorkflowRuns/styles.ts

```diff
@@ -19,6 +19,10 @@ const useStyles = makeStyles((theme) => ({
       color: theme.palette.primary.main,
       marginRight: theme.spacing(4.5),
     },
+
+    '& img': {
+      margin: theme.spacing(0, 0.25, -0.375, 0),
```

Comment:
is the negative margin required?

Reply:
For once yes, it is required as litmus-UI button won't allow me to push the img down without expanding the whole button height, I had to override that

Reply:
Cc: @arkajyotiMukherjee @S-ayanide